### ISSUE TITLE
problem with - Upgrading contao-themes-net/theme-components-bundle (1.3.8 => 1.3.9)

### ISSUE THREAD

**Issue by magicsepp** [contao-themes-net/nature-theme-bundle#44]
After successfull upgrade I run into following problem:
```[2022-03-27 12:00:13] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Compile Error: Declaration of ContaoThemesNet\NatureThemeBundle\Element\SliderElementNature::compile() must be compatible with ContaoThemesNet\ThemeComponentsBundle\Element\SliderElement::compile(): void" at D:\laragon4\www\cto44\vendor\contao-themes-net\nature-theme-bundle\src\Element\SliderElementNature.php line 18 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Compile Error: Declaration of ContaoThemesNet\\NatureThemeBundle\\Element\\SliderElementNature::compile() must be compatible with ContaoThemesNet\\ThemeComponentsBundle\\Element\\SliderElement::compile(): void at D:\\laragon4\\www\\cto44\\vendor\\contao-themes-net\\nature-theme-bundle\\src\\Element\\SliderElementNature.php:18)"} []```
downgrade to theme-components-bundle 1.3.8 fix the problem

**Comment by landzone21** [comment]
I have the same problem with Contao 4.9.28 and Theme-Bundle 1.8:
`Compile Error: Declaration of ContaoThemesNet\NatureThemeBundle\Element\SliderElementNature::compile() must be compatible with ContaoThemesNet\ThemeComponentsBundle\Element\SliderElement::compile(): void`
But downgrade to 1.3.8 didn't fix the problem!

**Comment by netzarbeiter** [comment]
**Exception**
```
Symfony\Component\ErrorHandler\Error\ FatalError
in vendor/contao-themes-net/nature-theme-bundle/src/Element/SliderElementNature.php (line 18)
``` 

**Stack Trace**
```
Symfony\Component\ErrorHandler\Error\FatalError:
Compile Error: Declaration of ContaoThemesNet\NatureThemeBundle\Element\SliderElementNature::compile() must be compatible with ContaoThemesNet\ThemeComponentsBundle\Element\SliderElement::compile(): void
at vendor/contao-themes-net/nature-theme-bundle/src/Element/SliderElementNature.php:18
```

**Temporary solution (works also with Contao 4.9.28)**

Add the following lines to composer.json

```json
"conflict": {
    "contao-themes-net/theme-components-bundle": "1.3.9"
},
```

**Comment by seibtph** [comment]
fixed in nature theme version 1.8.2
